### DISCUSSION TITLE
Cannot verify my github student, errors out after uploading the proof

### DISCUSSION THREAD

**Discussion by junbug01** [community/community#72623] [GitHub Education] [answered]
### Hi everyone,

Hey,

I am trying to claim my student discount using my school email address, but it asks for Upload proof of your academic status. When I upload and click on process my application it errors out and doesn't do anything! Please help me out.
![image](https://github.com/community/community/assets/87606844/7bae9f2a-481f-4148-9f6c-aba7d053a10e)

I have been facing this issue from past 2 months, please help

**Comment by andrej-kocijan** [comment, marked as answer]
I would suggest troubleshooting with the support guides [here](https://support.github.com/features/education), if nothing works, support can be contacted on the same link at the bottom of the page. Good luck!

**Reply by junbug01** [reply]
I have tried tried using the support guides, doesn't work
